### DISCUSSION TITLE
Osu Lazer crashing on android with Multi-Threading

### DISCUSSION THREAD

**Discussion by KrispyKebaeb** [ppy/osu#16781] [Q&A] [answered]
Whenever i change apps and leave osu open in background, and open it again, it crashes, it crashes even if i go to the app switcher for a second and go back to the app, i dont know exactly why does it happen but i think its because android switches osu to a single thread to make cpu space for other apps, while osu still trying to stay multi threaded, which leads to a crash.

**Comment by NotGumballer91** [comment]
[Please provide logs so they can look into the problem](https://github.com/ppy/osu/discussions/15188)

**Comment by Theighlin** [comment, marked as answer]
#16512
